core/app/core/src/lib/fields/int/templates/edit/int.component.ts

```diff
@@ -0,0 +1,87 @@
+/**
+ * SuiteCRM is a customer relationship management program developed by SalesAgility Ltd.
+ * Copyright (C) 2023 SalesAgility Ltd.
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Affero General Public License version 3 as published by the
+ * Free Software Foundation with the addition of the following permission added
+ * to Section 15 as permitted in Section 7(a): FOR ANY PART OF THE COVERED WORK
+ * IN WHICH THE COPYRIGHT IS OWNED BY SALESAGILITY, SALESAGILITY DISCLAIMS THE
+ * WARRANTY OF NON INFRINGEMENT OF THIRD PARTY RIGHTS.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * In accordance with Section 7(b) of the GNU Affero General Public License
+ * version 3, these Appropriate Legal Notices must retain the display of the
+ * "Supercharged by SuiteCRM" logo. If the display of the logos is not reasonably
+ * feasible for technical reasons, the Appropriate Legal Notices must display
+ * the words "Supercharged by SuiteCRM".
+ */
+
+import { isEqual } from 'lodash-es';
+import {Component} from '@angular/core';
+import {FormControl} from '@angular/forms';
+import {BaseNumberComponent} from '../../../base/base-number.component';
+import {SystemConfigStore} from '../../../../store/system-config/system-config.store';
+import {DataTypeFormatter} from '../../../../services/formatters/data-type.formatter.service';
+import {UserPreferenceStore} from '../../../../store/user-preference/user-preference.store';
+import {FieldLogicManager} from '../../../field-logic/field-logic.manager';
+import {FieldLogicDisplayManager} from '../../../field-logic-display/field-logic-display.manager';
+
+@Component({
+    selector: 'scrm-int-edit',
+    templateUrl: './int.component.html',
+    styleUrls: []
+})
+export class IntEditFieldComponent extends BaseNumberComponent {
+    public intValue = 0;
+    public intValueFormControl: FormControl = new FormControl(this.intValue);
```

Comment:
Hi @mpuyosa91, why do we need a separete formControl? why not use the one that already exists on the field?